TSC-Charter.md

```diff
@@ -92,45 +85,24 @@ including:
 * Mediating technical conflicts between Collaborators or Foundation
 projects.
 
-The TSC will define Node.js Foundation’s release vehicles and serve as
-Node.js Foundation’s primary technical liaison body with external open
-source projects, consortiums and groups.
+The TSC will define Node.js project’s release vehicles.
 
-## Section 6. Node.js Foundation Operations.
+## Section 5. Node.js Project Operations.
 
-The TSC will establish and maintain a development process for Node.js
-Foundation Projects. The development process will establish guidelines
+The TSC will establish and maintain a development process for the Node.js
+project. The development process will establish guidelines
 for how the developers and community will operate. It will, for example,
 establish appropriate timelines for TSC review (e.g. agenda items must be
 published at least a certain number of hours in advance of a TSC
 meeting).
 
-There will be multiple Projects under the Node.js Foundation organized by
-modules or subsystems. The TSC is responsible for organizing the Project
-structure, including possibly the creation and alignment of sub-Projects.
-Each Project must be within such policies as may be set by the Board,
-have a well-defined scope and must work within that scope. The
-development process will provide for Projects to follow the lifecycle
-process as described in the Project Lifecycle document. The development
-process will include a process for the TSC to oversee and approve changes
-in the lifecycle of a Project, which will include consideration of the
-following criteria:
-
-* Cleanliness of code base
-* Ample and diverse Contributors and Collaborators to assure vitality of
-the project.
-* Stability (e.g. presence of test suites, stable APIs and use of an
-  appropriate source-code control system).
-* Predictability of releases
-* Alignment with Node.js Foundation’s goals and priorities.
-
 The TSC and entire technical community will follow any processes as may
-be specified by the Board relating to the intake and license compliance
-review of contributions, including the Node.js Foundation IP Policy.
+be specified by the OpenJS Foundation Board relating to the intake and license compliance
+review of contributions, including the OpenJS Foundation IP Policy.
```

Comment:
Is there an OpenJS Foundation IP Policy currently defined? If so, can this link to it?

Reply:
@brianwarner is there a link for that?

Reply:
@mhdawson and @Jonahss - I'm working on getting these various documents updated.  I think it's best to leave it blank for now, and I'll be back with a link once we have something final which we can link.

Reply:
Ok, this should land once we get the governance update landed (7 days from now) and then we can open a PR once the links are ready.